---
layout: default
title: alkan lab / support
categories:
 - support
---

        <h4> <b>Integrated approaches for genomic variation discovery using
            high throughput sequencing</b></h4>
        <ul>
          <ul>
            <li>European Union Marie Curie Actions Career Integration Grant
              (PCIG10-GA-2011-303772), 2012-2016</li>
            <li>PI: Can Alkan</li>
            <li>Students: Marzieh Eslami Rasekh, Can Fırtına</li>
            <li><span class="st">Total €</span>100,000 for four years.</li>
            <li>The goal of this project is to develop computational methods to
              understand genomic variation using high throughput sequencing
              (HTS) with a special focus on structural variation (SV) including
              copy-number variation (CNV) and balanced rearrangements
              (inversions, translocations) in the complex regions of the human
              genome that are rich in repeats and duplications.<br />
            </li>
          </ul>
        </ul>
        <span style="font-weight: bold;">Abstract</span><br />
        <br />
        The new sequencing technologies revolutionize genomics as they promise
        low-cost, high-throughput sequencing of both new species and different
        individuals to more fully analyze the patterns of genetic variation.
        These “next-generation” platforms started to contribute our
        understanding of human genome diversity with the 1000 Genomes Project
        that employs the high throughput sequencing (HTS) methods to produce the
        most detailed map of human variation. Other large scale sequencing
        projects are initiated to characterize genomes to assess characteristics
        of human genome diversity, to find genetic causes for disease, and infer
        the evolutionary history of species. Although we can now generate data
        at a rate previously unimaginable, the analysis of such data is
        lingering as currently available algorithms to analyze HTS data show
        different strengths and biases for different classes of variation. There
        is a need to forge an alliance between computer science and genomics to
        devise better methods to use the massive amount of sequence data. Here
        we propose to develop novel algorithms to comprehensively and quickly
        discover all forms of genomic variants including point mutations, indel
        polymorphisms and structural variation while resolving inconsistencies
        among different variants to accurately identify normal and
        disease-causing variation.<br />
        <br />
        <h4>Dissemination</h4>
        <ul>
          <li><a href="http://www.ncbi.nlm.nih.gov/pubmed/23369189"><strong>Accelerating



                read mapping with FastHASH.</strong></a> Hongyi Xin, Donghyuk
            Lee, Farhad Hormozdiari, Samihan Yedkar, Onur Mutlu, <span style="color:             rgb(0, 0, 0);"><strong>Can
                Alkan</strong></span>. <em>BMC Genom</em><em>ics</em>, 14(Suppl
            1):S13, 2013</li>
          <li><a href="http://www.ncbi.nlm.nih.gov/pubmed/24453982"><strong>Genome
                Sequencing Highlights the Dynamic Early History of Dogs.</strong></a>
            Adam H. Freedman, Ilan Gronau, Rena M. Schweizer, Diego  Ortega-Del
            Vecchyo, Eunjung Han, Pedro M. Silva, Marco Galaverni, Zhenxin Fan,
            Peter Marx, Belen Lorente-Galdos, Holly Beale, Oscar Ramirez, Farhad
            Hormozdiari, <strong style="color: black;">Can Alkan</strong>,
            Carles Vilà, Kevin Squire, Eli Geffen, Josip Kusak, Adam R. Boyko,
            Heidi G. Parker, Clarence Lee, Vasisht Tadigotla, Adam Siepel,
            Carlos D. Bustamante, Timothy T. Harkins, Stanley F. Nelson, Elaine
            A. Ostrander, Tomas Marques-Bonet, Robert K. Wayne, John Novembre. <em>PLoS
              Genetics</em>, 10(1): e1004016, 2014.</li>
          <li><a href="http://www.ncbi.nlm.nih.gov/pubmed/24764354"><span style="color: #286ea0;"><span
                  style="font-weight: bold;">Early postzygotic mutations
                  contribute to de novo variation in a healthy monozygotic twin
                  pair.</span></span></a> Gülşah M Dal, Bekir Ergüner, Mahmut S
            Sağıroğlu, Bayram Yüksel, Onur Emre Onat, <strong style="color: black;">Can
              Alkan</strong>, Tayfun Özçelik. <em>J Med Genet</em>,
            51(7):455-459,<cite><span itemprop="datePublished" class="slug-pub-date"></span><span
                class="slug-vol"><span class="cit-sep cit-sep-after-article-vol"></span></span><span
                class="slug-pages"> </span></cite>2014.</li>
          <li><strong><a href="http://www.ncbi.nlm.nih.gov/pubmed/24810850">mrsFAST-Ultra:
                a compact, SNP-aware mapper for high performance sequencing
                applications.</a></strong> Faraz Hach*, Iman Sarrafi*, Farhad
            Hormozdiari, <strong style="color: black;">Can Alkan</strong>, Evan
            E. Eichler, S. Cenk Sahinalp. <em>Nucl Acids Res,</em> Jul;42(Web
            Server issue):W494-500, 2014.</li>
          <li><strong><strong><strong><a href="http://www.ncbi.nlm.nih.gov/pubmed/25461772">Fast
                    and accurate mapping of Complete Genomics reads.</a> </strong></strong></strong>Donghyuk
            Lee*, Farhad Hormozdiari*, Hongyi Xin, Faraz Hach, Onur Mutlu, <span
              style="font-weight: bold; color: black;">Can Alkan</span>. <em>Methods</em>,
            Jun;79-80:3-10, 2015.<strong><strong><br />
              </strong></strong></li>
          <li><strong><a href="http://www.ncbi.nlm.nih.gov/pubmed/25577434">Shifted
                Hamming Distance: a fast and accurate SIMD-friendly filter to
                accelerate alignment verification in read mapping.</a> </strong>Hongyi
            Xin, John Greth, John Emmons, Gennady Pekhimenko, Carl Kingsford, <strong><span
                style="color: #070c07;">Can Alkan</span></strong>* and Onur
            Mutlu*. <em>Bioinformatics, </em>[published online, Jan 10], 2015.</li>
          <li><strong><a href="http://www.ncbi.nlm.nih.gov/pubmed/26568624"><strong></strong></a><strong><a
                  href="http://www.ncbi.nlm.nih.gov/pubmed/26568624">Optimal
                  Seed Solver: Optimizing Seed Selection in Read Mapping.</a></strong></strong>
            Hongyi Xin, Sunny Nahar, Richard Zhu, John Emmons, Gennady
            Pekhimenko, Carl Kingsford, <span style="color: black; font-weight: bold;">Can
              Alkan*</span>, Onur Mutlu*<em>.</em> <em>Bioinformatics</em>, Jun
            1;32(11):1632-42, 2016.</li>
          <li><strong><strong><strong><a href="http://www.ncbi.nlm.nih.gov/pubmed/26382624">Robustness
                    of massively parallel sequencing platforms</a>. </strong></strong><span
                style="color: black;">Pınar Kavak</span></strong>, Bayram
            Yüksel, Soner Aksu, M. Oğuzhan Külekçi, Tunga Güngör, Faraz Hach, S.
            Cenk Sahinalp, Turkish Human Genome Project, <strong style="color: black;">Can
              Alkan*</strong>, M. Şamil Sağıroğlu*. <em>PLoS ONE</em>, Sep
            18;10(9):e0138259, 2015.</li>
          <li><strong><strong><strong><a href="http://www.ncbi.nlm.nih.gov/pubmed/26432245">A
                    global reference for human genetic variation.</a> </strong></strong></strong><span
              class="cit-sep cit-sep-separator"><span class="cit-sep               cit-sep-separator">1000
                Genomes Project Consortium. <span style="font-style: italic;">Nature</span>,</span></span><strong><strong><strong>
                </strong></strong></strong>Oct<span class="cit-sep cit-sep-separator"><span
                class="cit-sep               cit-sep-separator"> 1; </span></span>526
            (7571):68–74, 2015.</li>
          <li><strong> <a href="https://www.ncbi.nlm.nih.gov/pubmed/27153582">On
                genomic repeats and reproducibility</a>. <span style="color: black;">Can
                Firtina</span></strong> and <span style="color: black; font-weight: bold;">Can
              Alkan</span>. <span style="font-style: italic;">Bioinformatics</span>,
            Aug 1;32(15):2243-7, 2016.</li>
          <li><a href="https://www.ncbi.nlm.nih.gov/pubmed/28073353"><strong>Discovery
                of large genomic inversions using long range information.</strong></a>
            <strong style="color: black;">Marzieh Eslami Rasekh</strong>,
            Giorgia Chiatante, Mattia Miroballo, Joyce Tang, Mario Ventura,
            Chris T. Amemiya, Evan E. Eichler, Francesca Antonacci*, <strong style="color: black;">Can
              Alkan</strong>*. <em>BMC Genomics</em>, Jan 10;18(1):65, 2017.</li>
          <li><strong><a href="http://ipsitransactions.org/journals/papers/tir/2017july/p5.php">MAGNET:
                understanding and improving the accuracy of genome pre-alignment
                filtering</a></strong>. <strong style="color: black;">Mohammed
              Alser</strong>, Onur Mutlu*, <strong style="color: black;">Can
              Alkan</strong>*. IPSI Transactions on Internet Research, 13(2),
            2017.</li>
          <li><strong><a href="https://www.ncbi.nlm.nih.gov/pubmed/28881988"> </a></strong><strong><a
                href="https://www.ncbi.nlm.nih.gov/pubmed/28575161"><strong>GateKeeper:
                  a new hardware architecture for accelerating pre-alignment in
                  DNA short read mapping.</strong></a> <strong style="color: black;">Mohammed
                Alser</strong></strong>,<strong> </strong>Hasan Hassan, Hongyi
            Xin, Oguz Ergin, Onur Mutlu*,<strong> <strong style="color: black;">Can
                Alkan</strong>*</strong>. <em>Bioinformatics</em>, Nov 1;
            33(21):3335-63, 2017.<strong></strong></li>
          <li>
            <div class="list-title mathjax"><strong></strong><a href="https://www.ncbi.nlm.nih.gov/pubmed/30124947"><strong>Hercules:
                  a profile HMM-based hybrid error correction algorithm for long
                  reads.</strong></a><strong> <span style="color: black;">Can
                  Firtina</span></strong><span style="color: black;"></span>,<strong>
              </strong>Ziv-Bar Joseph,<strong> <span style="color: black;">Can
                  Alkan*</span>, </strong>A. Ercument Cicek*. <em>Nucleic
                Acids Research</em>, 46(21): e125, 2018.</div>
          </li>
        </ul>
