---
layout: default
title: alkan lab / software
categories:
 - software
---

        <h3>Official GitHub repository:</h3>
        <ul>
          <li><a href="https://github.com/BilkentCompGen">https://github.com/BilkentCompGen</a></li>
        </ul>
        <h3>Docker Hub repository</h3>
        <ul>
          <li><a href="https://hub.docker.com/u/alkanlab/">https://hub.docker.com/u/alkanlab/</a></li>
        </ul>
        <h3>High Throughput sequencing</h3>
        <ul>
          <li><a style="font-weight: bold;" href="https://github.com/BilkentCompGen/mrfast">mrFAST</a><span
              style="font-weight: bold;">:</span> Read mapper developed
            specifically for structural variation and segmental duplication
            detection using the Illumina platform. Supports small indel
            alignments. </li>
          <li> <a style="font-weight: bold;" href="https://github.com/sfu-compbio/mrsfast">mrsFAST</a><span
              style="font-weight: bold;">:</span> Similar to mrFAST, Hamming
            distance only. </li>
          <li><span style="font-weight: bold;"> <a href="https://github.com/BilkentCompGen/mrcanavar">mrCaNaVaR</a>:
            </span>Read depth analysis method to characterize segmental
            duplications and predict absolute copy numbers.<span style="font-weight: bold;"><br />
            </span> </li>
          <li><span style="font-weight: bold;"><a href="https://github.com/sfu-compbio/scalce">SCALCE</a>:
            </span>Tool to efficiently compress FASTQ files.</li>
          <li><strong><a href="https://github.com/BilkentCompGen/tardis">TARDIS</a>:</strong>
            Toolkit for automatic and rapid discovery of structural variants.</li>
          <li><strong><a href="https://github.com/vpc-ccg/pamir">Pamir</a></strong>:
            Discovery and genotyping of novel sequence insertions in many
            sequenced individuals.</li>
            <!--
          <li><a href="https://github.com/BilkentCompGen/lava"><strong>LaVa</strong></a>:
            Large Variation discovery using hybrid sequence data.</li> -->
          <li><strong><a href="https://github.com/BilkentCompGen/valor">VALOR2</a></strong>:
            Discover inversions using long range information -- linked-read
            version.</li>
          <li><strong><a href="https://github.com/BilkentCompGen/GateKeeper">GateKeeper</a></strong>:
            FPGA design for accelerating pre-alignment in DNA short read
            mapping.</li>
          <li><strong><a href="https://github.com/BilkentCompGen/GateKeeper-GPU">GateKeeper-GPU</a></strong>:
            GPGPU design for accelerating pre-alignment in DNA short read
            mapping.</li>
          <li><strong><a href="https://github.com/BilkentCompGen/MAGNET">MAGNET</a></strong>:
            FPGA-based filtering strategy with high accuracy across different
            edit distance thresholds.</li>
          <li><strong><a href="https://github.com/CMU-SAFARI/Shouji">Shouji</a></strong>:
            Fast and efficient computation of banded sequence alignment.</li>
          <li><a href="https://github.com/CMU-SAFARI/SneakySnake"><strong>SneakySnake</strong></a>:
            Approximate alignment computation on CPU, GPU, and FPGA by reducing
            alignment problem to single net routing problem.</li>
          <li><a href="https://github.com/BilkentCompGen/hercules"><strong>Hercules</strong></a>:
            Profile HMM-based hybrid error correction algorithm for long reads.</li>
          <li><a href="https://github.com/CMU-SAFARI/Apollo"><strong>Apollo</strong></a>:
            Profile HMM-based genome assembly polishing tool.</li>
          <li><a href="https://github.com/vpc-ccg/rinslr"><strong>RinsLR</strong></a>:
            Discovery of mid range novel sequence insertions using long-read
            sequencing.</li>
          <li><a href="https://github.com/ciceklab/DECoNT"><strong>DeCoNT</strong></a>:
            CNV prediction polisher on whole exome sequencing data.</li>
          <li><a href="https://github.com/CMU-SAFARI/GenASM"><strong>GenASM</strong></a>:
            approximate string matching (ASM) acceleration framework for genome sequence analysis.</li>
          <li><a href="https://github.com/CMU-SAFARI/SeGraM"><strong>SeGraM</strong></a>:
             universal genomic mapping accelerator that supports both sequence-to-graph mapping and sequence-to sequence mapping, for both short and long reads.</li>
          <li><a href="https://github.com/CMU-SAFARI/FastRemap"><strong>FastRemap</strong></a>:
            Tool for quickly remapping reads between genome assemblies.</li>
          <li><a href="https://github.com/CMU-SAFARI/BLEND"><strong>BLEND</strong></a>:
            Fast, memory-efficient, and accurate mechanism to find fuzzy seed matches in genome analysis.</li>
          <li><a href="https://github.com/CMU-SAFARI/AirLift"><strong>AirLift</strong></a>:
            Fast and comprehensive technique for translating alignments between reference genomes.</li>
          <li><a href="https://github.com/CMU-SAFARI/TargetCall"><strong>TargetCall</strong></a>:
            Pre-basecalling filter for nanopore sequencing.</li>
          <li><a href="https://github.com/BilkentCompGen/lcpan"><strong>LCPan</strong></a>:
            Variation graph construction using Locally Consistent Parsing.</li>            
        </ul>
        <h3>Others</h3>
        <ul>
          <li><span style="font-weight: bold;"><a href="https://github.com/BilkentCompGen/mam">MaM</a>:
            </span>A software tool that processes and manipulates multiple
            alignments of genomic sequences. </li>
          <li><a href="https://github.com/0xTCG/biser"><strong>BISER</strong></a>:
            Characterization of segmental duplication evolutionary structure
            across  genome assemblies.</li>
            <li><a href="https://github.com/BilkentCompGen/brifin"><strong>BriFin</strong></a>:
            Tool designed to detect specialized hubs among the proteins providing cell-to-cell interactions in protein-protein interaction networks that include the proteins of two interacting cells.</li>
            <li><a href="https://github.com/BilkentCompGen/saber"><strong>SABER</strong></a>:
            Tool to perform pairwise sequence alignment using block edits and rearrangements.</li>
	    <li><a href="https://github.com/BilkentCompGen/linden"><strong>Linden</strong></a>:
            Alternating Lyndon Factorization Tree data structure and its application to whole-genome alignments.</li>
	    <li><a href="https://github.com/BilkentCompGen/lcptools"><strong>LCPtools</strong></a>:
            Implementation of the Locally Consistent Parsing (LCP) algorithm for string processing as an API.</li>
	    <li><a href="https://github.com/BilkentCompGen/gencore"><strong>GenCore</strong></a>:
            Fast alignment-free genomic distance estimation using LCP.</li>
        </ul>
        
        <h3>Deprecated Tools</h3>
        <ul>
         <li><h4>High Throughput Sequencing <br /></h4>
        <ul>
                  <li> <a style="font-weight: bold;" href="https://github.com/BilkentCompGen/drfast">drFAST</a><span
              style="font-weight: bold;">:</span> Read mapper for di-base
            color-space reads generated with the SOLiD platform. </li>
          <li><a style="font-weight: bold;" href="https://github.com/BilkentCompGen/sirfast">sirFAST</a><span
              style="font-weight: bold;">:</span> Read mapper for short
            interrupted reads generated with the Complete Genomics platform. </li>
          <li> <span style="font-weight: bold;"><a href="https://variationhunter.sourceforge.net">VariationHunter</a>:</span>
            Structural variation calling algorithm using read pair mapping
            information including suboptimal alignments. (replaced by <a href="https://github.com/BilkentCompGen/tardis">TARDIS</a>)</li>
          <li> <span style="font-weight: bold;"><a href="https://splitread.sourceforge.net">SPLITREAD</a>:
            </span>Detection of structural variants and indels from genome and
            exome sequencing data. </li>
          <li> <span><strong><a href="http://novelseq.sourceforge.net">NovelSeq</a></strong>:
            </span>Novel sequence insertion discovery framework. (replaced by <a href="https://github.com/vpc-ccg/pamir">Pamir</a>) </li>
          <li><strong><a href="https://github.com/BilkentCompGen/valor">VALOR</a>:</strong>
            Discover inversions using long range information.(replaced by <a href="https://github.com/BilkentCompGen/valor">VALOR2</a>)</li>

          </ul></li>
        <li><h4>RNA analysis <br /></h4>
        <ul>
          <li> <span style="font-weight: bold;">taveRNA suite</span></li>
          <ul>
            <li><span style="font-weight: bold;"><a href="">inteRNA</a>:
              </span>Algorithm that predicts the joint secondary structure of
              two RNA sequences.<span style="font-weight: bold;"><br />
              </span></li>
            <li><span style="font-weight: bold;"><a href="">alteRNA</a>:
              </span>Densityfold algorithm that<span style="font-weight: bold;">
              </span>minimizes a linear combination of energy density and the
              total free energy.  </li>
            <li><span style="font-weight: bold;"><a href="">pRuNA</a>:
              </span>RNA interaction search engine. pRuNA is a sequence based
              pruning technique that eliminates a significant fraction of a non
              coding RNA (ncRNA) data set and retains only the most likely ncRNA
              candidates for forming a stable joint structure with the query
              mRNA.<br />
            </li>
            <li><a href=""><span style="font-weight: bold;">piRNA:</span></a><span
                style="font-weight: bold;"> </span>RNA-RNA interaction
              partition function algorithm to predict the joint partition
              function, equilibrium concentration, ensemble energy, and melting
              temperature for two RNA sequences.<br />
              </li>
            <li><span style="font-weight: bold;"><a href="">smyRNA</a>:
              </span>Novel <span style="font-style: italic;">ab initio</span>
              non-coding RNA finder.<span style="font-weight: bold;"><br />
              </span></li>
          </ul></li>
          <li><h4>Others<br /></h4>
        <ul>
                  <li><a href="https://github.com/BilkentCompGen/repeatnet"><span style="font-weight: bold;">RepeatNet:</span></a>
            An <span style="font-style: italic;">ab initio</span> centromeric
            sequence detection algorithm.</li>
          <li><a href="https://github.com/vpc-ccg/sedef/"><strong>SEDEF</strong></a>:
            Characterization of segmental duplications within genome assemblies. (replaced by <a href="https://github.com/0xTCG/biser">BISER</a>)</li>

                  </ul></li>
        </ul>
	</ul>
