---
layout: default
title: alkan lab / support
categories:
 - support
---     

        <h4>Novel algorithms and hardware designs for ultra-fast next-gen
          sequence analysis<br />
        </h4>
        <ul>
          <ul>
            <li>United States of America National Institutes of Health (R01
              HG006004), 2011-2015 <br />
            </li>
            <li>PI: Onur Mutlu</li>
            <li>Co-PI: Can Alkan</li>
            <li>Subaward amount (4 years): $462,847<br />
            </li>
            <li>The goal of this project is to develop specialized hardware
              architectures to accelerate mapping reads generated with the high
              throughput sequencing platforms.</li>
          </ul>
        </ul>
        <h4>People</h4>
        <ul>
          <ul>
            <li>Principal Investigators: Assistant Prof. Can Alkan (Bilkent U.)
              and Assistant Prof. Onur Mutlu (Carnegie Mellon U.)</li>
            <li>Students</li>
            <ul>
              <li>CMU: Hongyi Xin, Donghyuk Lee, Samihan Yedkar, Damla Şenol
                Çalı</li>
              <li>UCLA: Farhad Hormozdiari</li>
              <li>Bilkent: Mustafa Korkmaz, Azita Nouri, Mohammed Alser, Tuğba
                Doğan<br />
              </li>
            </ul>
          </ul>
        </ul>
        <h4>Abstract</h4>
        <br />
        Our proposed research aims to accelerate next generation sequence
        analysis 1000-fold or more by combining our knowledge in genomic
        sequence analysis, algorithms development, and computer
        architecture/engineering. Our plan to address the problems of processing
        unprecedented amounts of sequence data has three major components.
        First, we will develop and improve sophisticated software algorithms and
        tools to handle large amounts of sequence reads generated by all major
        NGS platforms without sacrificing sensitivity while correcting for the
        sequencing biases associated by each of the NGS platforms. Our
        algorithms will also be able to map reads in the duplicated regions of
        the genome and report the underlying sequence variation, an important
        feature especially to characterize segmental duplications and structural
        variation that no other read mapping tool can currently achieve. Second,
        we will boost the performance and efficiency of our algorithms (100 to
        1000-fold) by accelerating the required inherently-parallel computations
        of the sequence search problem on massively-parallel hardware engines
        available today, graphics processing units (GPUs). Finally, we will
        design specialized hardware architectures to enhance the speed of
        sequence analysis beyond orders of magnitude while reducing energy
        consumed by it by 100-fold or more.<br />
        <br />
        <h4>Dissemination</h4>
        <ul>
          <li><span class="cit-sep cit-sep-separator"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23047557"><span
                  style="font-weight: bold;">SCALCE: boosting Sequence
                  Compression Algorithms using Locally Consistent Encoding.</span></a>
              Faraz Hach, <span itemprop="name" class="name"><span class="name-search">Ibrahim
                  Numanagić</span></span>, <span style="font-weight: bold; color:               black;">Can
                Alkan</span>, S. Cenk Sahinalp. <span style="font-style: italic;">Bioinformatics</span>,
              Dec</span><span class="cit-sep cit-sep-separator"><span class="cit-sep cit-sep-separator"><span
                  class="cit-sep                 cit-sep-separator">
                  1;28(23):3051-57, 2012.</span></span></span><a href="http://www.ncbi.nlm.nih.gov/pubmed/23369189"><strong></strong></a></li>
          <li><a href="http://www.ncbi.nlm.nih.gov/pubmed/23369189"><strong>Accelerating



                read mapping with FastHASH.</strong></a> Hongyi Xin, Donghyuk
            Lee, Farhad Hormozdiari, Samihan Yedkar, Onur Mutlu, <span style="color:             rgb(0, 0, 0);"><strong>Can
                Alkan</strong></span>. <em>BMC Genom</em><em>ics</em>, 14(Suppl
            1):S13, 2013.</li>
          <li><strong><a href="http://www.ncbi.nlm.nih.gov/pubmed/24810850">mrsFAST-Ultra:
                a compact, SNP-aware mapper for high performance sequencing
                applications.</a></strong> Faraz Hach*, Iman Sarrafi*, Farhad
            Hormozdiari, <strong style="color: black;">Can Alkan</strong>, Evan
            E. Eichler, S. Cenk Sahinalp. <em>Nucl Acids Res,</em> Jul;42(Web
            Server issue):W494-500, 2014.</li>
          <li><strong><a href="http://www.ncbi.nlm.nih.gov/pubmed/25461772">Fast
                and accurate mapping of Complete Genomics reads.</a> </strong>Donghyuk
            Lee*, Farhad Hormozdiari*, Hongyi Xin, Faraz Hach, Onur Mutlu, <span
              style="font-weight: bold; color: black;">Can Alkan</span>. <em>Methods</em>,
            [epub October 22], doi :10.1016/j.ymeth.2014.10.012, 2014.</li>
          <li><strong><a href="http://www.ncbi.nlm.nih.gov/pubmed/25577434">Shifted
                Hamming Distance: a fast and accurate SIMD-friendly filter to
                accelerate alignment verification in read mapping.</a> </strong>Hongyi
            Xin, John Greth, John Emmons, Gennady Pekhimenko, Carl Kingsford, <strong><span
                style="color: #070c07;">Can Alkan</span></strong>* and Onur
            Mutlu*. <em>Bioinformatics, </em>[published online, Jan 10], 2015.</li>
          <li><strong><a href="http://www.ncbi.nlm.nih.gov/pubmed/26568624">Optimal
                Seed Solver: Optimizing Seed Selection in Read Mapping.</a> </strong>Hongyi
            Xin, Sunny Nahar, Richard Zhu, John Emmons, Gennady Pekhimenko, Carl
            Kingsford, <span style="color: black; font-weight: bold;">Can
              Alkan*</span>, Onur Mutlu*<em>.</em><strong> </strong><em>Bioinformatics</em>,
            Jun 1;32(11):1632-42, 2016.</li>
          <li><strong><a href="http://ipsitransactions.org/journals/papers/tir/2017july/p5.php">MAGNET:
                understanding and improving the accuracy of genome pre-alignment
                filtering</a></strong>. <strong style="color: black;">Mohammed
              Alser</strong>, Onur Mutlu*, <strong style="color: black;">Can
              Alkan</strong>*. IPSI Transactions on Internet Research, 13(2),
            2017.</li>
          <li><strong><a href="https://www.ncbi.nlm.nih.gov/pubmed/28881988"> </a></strong><strong><a
                href="https://www.ncbi.nlm.nih.gov/pubmed/28575161"><strong>GateKeeper:
                  a new hardware architecture for accelerating pre-alignment in
                  DNA short read mapping.</strong></a> <strong style="color: black;">Mohammed
                Alser</strong></strong>,<strong> </strong>Hasan Hassan, Hongyi
            Xin, Oguz Ergin, Onur Mutlu*,<strong> <strong style="color: black;">Can
                Alkan</strong>*</strong>. <em>Bioinformatics</em>, Nov 1;
            33(21):3335-63, 2017.<strong></strong></li>
          <li><strong><a href="https://www.ncbi.nlm.nih.gov/pubmed/29617724"> </a><a
                href="https://www.ncbi.nlm.nih.gov/pubmed/29764378"><strong>GRIM-Filter:
                  fast seed location filtering in DNA read mapping using
                  processing-in-memory technologies.</strong></a> </strong>Jeremie
            S. Kim, Damla Senol Cali, Hongyi Xin, Donghyuk Lee, Saugata Ghose, <strong><span
                style="color: black;">Mohammed Alser</span></strong>, Hasan
            Hassan, Oguz Ergin, <span><span style="color: black;"><strong>Can
                  Alkan</strong>*</span></span>, Onur Mutlu*. <em>BMC Genomics,
            </em>19 (Suppl 2):89, 2018.</li>
          <li><strong><a href="https://www.ncbi.nlm.nih.gov/pubmed/29617724">Nanopore
                sequencing technology and tools for genome assembly:
                computational analysis of the current state, bottlenecks and
                future directions.</a></strong><span style="color: #286ea0;"> </span>Damla
            Senol Cali, Jeremie S. Kim, Saugata Ghose, <span><strong style="color: black;">Can
                Alkan*</strong></span>, Onur Mutlu*. <em>Briefings in
              Bioinformatics</em>, [epub Apr 2; doi: 10.1093/bib/bby017], 2018.</li>
        </ul>
