---
layout: default
title: alkan lab / support
categories:
 - support
---

        <h4>Development and application of computational methods to analyze next
          generation sequence data to characterize both normal and disease
          causing variation, and build <span style="font-style: italic;">de
            novo</span> genome assemblies</h4>
        <ul>
          <ul>
            <li>European Molecular Biology Organization Installlation Grant
              (IG-2521), 2013-2016<br />
            </li>
            <li>PI: Can Alkan</li>
            <li>Students: Can Fırtına, Halil İbrahim Özercan, Ezgi Ebren, Zülal
              Bingöl, Balanur İçen</li>
            <li><span class="st">Total €</span>150,000 for three years.</li>
            <li>This is an EMBO Installation Grant that is given to young
              researchers to install their independent laboratories. The funds
              of this project will be used in various projects within the Lab
              for Bioinformatics and Computational Genomics that involves
              development of computational methods to utilize high throughput
              sequencing (HTS) for different applications.<br />
            </li>
          </ul>
        </ul>
        <span style="font-weight: bold;">Abstract</span><br />
        <br />
        My main research focus is the analysis of biological sequences generated
        with the next-gen sequencing platforms, and my plans for independent
        research are twofold: computational algorithm development, and
        application of methods to sequencing data to analyze genomic variation
        in both normal and diseased individuals. My long term goal is to
        accurately and comprehensively predict genomic variation from next
        generation datasets routinely.<br />
        <br />
        <h4>Dissemination</h4>
        <br />
        <ul>
          <li> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24764354"><span style="color: #286ea0;"><span
                  style="font-weight: bold;">Early postzygotic mutations
                  contribute to de novo variation in a healthy monozygotic twin
                  pair.</span></span></a> Gülşah M Dal, Bekir Ergüner, Mahmut S
            Sağıroğlu, Bayram Yüksel, Onur Emre Onat, <strong style="color: black;">Can
              Alkan</strong>, Tayfun Özçelik.  <em>J Med Genet</em>,
            51(7):455-459,<cite><span itemprop="datePublished" class="slug-pub-date"></span><span
                class="slug-vol"><span class="cit-sep cit-sep-after-article-vol"></span></span><span
                class="slug-pages"> </span></cite>2014.</li>
          <li><strong><a href="http://www.ncbi.nlm.nih.gov/pubmed/25376095">Whole
                genome sequencing of Turkish genomes reveals functional private
                alleles and impact of genetic interactions with Europe, Asia and
                Africa.</a>  <span style="color: black;">Can Alkan</span></strong>,
            Pınar Kavak, Mehmet Somel, Omer Gokcumen, Serkan Uğurlu, Ceren
            Saygı, <span style="font-weight: bold; color: black;">Elif Dal</span>,
            Kuyaş Buğra, Tunga Güngör, S Cenk Sahinalp, Nesrin Özören and
            Cemalettin Bekpen. <em>BMC Genomics, </em>15(1):963, <span class="pseudotab">doi:10.1186/1471-2164-15-963,
              </span>2014.</li>
          <li><strong><a href="https://www.ncbi.nlm.nih.gov/pubmed/28881988">Discovery
                and genotyping of novel sequence insertions in many sequenced
                individuals.</a></strong> <strong style="color: black;">Pınar
              Kavak</strong>, Yen-Yi Lin, Ibrahim Numanagić, Hossein Asghari,
            Tunga Güngör, <strong style="color: black;">Can Alkan</strong>*,
            Faraz Hach*. <em>Bioinformatics</em>,  Jul 15; 33 (14): i161-i169,
            2017.</li>
          <li><a href="https://www.ncbi.nlm.nih.gov/pubmed/30076130"><strong>Realizing
                the potential of blockchain technologies in genomics</strong></a><a
              href="https://www.ncbi.nlm.nih.gov/pubmed/30039363"><strong>.</strong></a><strong>
              <span style="color: black;">Halil Ibrahim Ozercan, Atalay Mert
                Ileri</span>, </strong>Erman Ayday<strong>, <span style="color: black;">Can
                Alkan</span></strong>. <em>Genome Research</em>, 28(9):<strong>
            </strong>1255-1263, 2018.</li>
          <li><strong><a href="https://www.ncbi.nlm.nih.gov/pubmed/29617724">Nanopore
                sequencing technology and tools for genome assembly:
                computational analysis of the current state, bottlenecks and
                future directions.</a></strong><span style="color: #286ea0;"> </span>Damla
            Senol Cali, Jeremie S. Kim, Saugata Ghose, <span><strong style="color: black;">Can
                Alkan*</strong></span>, Onur Mutlu*. <em>Briefings in
              Bioinformatics</em>, [epub Apr 2; doi: 10.1093/bib/bby017], 2018.</li>
          <li><a href="https://www.ncbi.nlm.nih.gov/pubmed/30423092"><strong>Fast
                characterization of segmental duplications in genome assemblies</strong></a><strong>.
            </strong>Ibrahim Numanagić, <strong style="color: black;">Alim S.
              Gökkaya</strong>, Lillian Zhang, Bonnie Berger, <strong style="color: black;">Can
              Alkan</strong><strong style="color: black;">*</strong>, Faraz
            Hach*. <em>Bioinformatics</em>, Sep 1; 34(17):i706-714, 2018.<strong></strong></li>
        </ul>
